File: app/src/main/java/com/example/repairservicesapp/app/AppManager.java
package com.example.repairservicesapp.app;

import com.example.repairservicesapp.model.User;

public class AppManager {

    public static AppManager instance = new AppManager();
    public User user = new User();

    private AppManager() { }

    public void setUser(User user) { this.user = user; }
}

File: app/src/main/java/com/example/repairservicesapp/app/AppManager.kt
package com.example.repairservicesapp.app

import com.example.repairservicesapp.model.Stats
import com.example.repairservicesapp.model.User

class AppManager private constructor() {
    @JvmField
    var user = User()
    @JvmField
    var stats = Stats()
    fun setUser(user: User) {
        this.user = user
    }

    fun setStats(stats: Stats) {
        this.stats = stats
    }

    companion object {
        @JvmField
        var instance = AppManager()
    }
}